Added bootstrap for styling

grudger007 opened the pull request:
Added bootstrap V4 css and js files for stylking and changed html files accordingly.

grudger007 committed:
Added bootstrap for styling

Added bootstrap V4 css and js files for stylking and changed html files accordingly.

grudger007 commented:
It says codecov for views.py changed?? I didn't even touch that file. Looking for suggestions how to pass this codecov.

kmcdon10 commented on imageprocessor/templates/base.html:
```diff
@@ -2,16 +2,44 @@
+		  <div class="collapse navbar-collapse" id="navbarNav">
+			<ul class="navbar-nav">
+			  <li class="nav-item active">
+				<a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
```
Since you have a logo that links to the home (root) page, you don't need a second link called home that also links to  the homepage

grudger007 replied:
removed home button

kmcdon10 commented on imageprocessor/templates/index.html:
```diff
@@ -1,18 +1,41 @@
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-3">
+              <a href = '{% url 'classify' %}' style="text-decoration: none !important;"><h3><span><img src="{% static 'img/cloud-upload.png' %}" width="100" height="100" alt=""></span> <br>Image Classifier</h3>
```
This would be easier to read if you used indentation instead of putting a bunch of nested tags all on the same line. You also don't need the wrap the image in a h3 or span tag. Instead of having a line break via <br/> you should do this with styling.

grudger007 replied:
added indentation and corrected tags alignment.

kmcdon10 commented on imageprocessor/templates/index.html:
```diff
@@ -1,18 +1,41 @@
+              <p class="lead mb-0">Upload multiple images to generate tags! <span class="glyphicon glyphicon-search"></span></p></a>
+          </div>
+          <div class="col-lg-3">
+              <a href = '{% url 'tagsearch' %}' style="text-decoration: none !important;"><h3><span><img src="{% static 'img/image-search.png' %}" width="100" height="100" alt=""></span><br>Tag Searcher</h3>
```
It's considered bad practice to use !important, it makes writing styles quickly become really unmanageable, and it looks like it's not needed here.

grudger007 replied:
Removed it from styling

EdRW committed:
Created .codecov.yml file to fix PR status issue

The overall coverage minimum is set to 80%
The difference threshold for status is set to 1%

EdRW commented on .codecov.yml:
```diff
@@ -0,0 +1,6 @@
+  status:
+    project:
+      default:
+        threshold: 1
```
I added this codecov.yml file to fix the issue of PRs failing if coverage drops by any amount. The threshold is set to 1%

grudger007 replied:
Thanks Ed

grudger007 committed:
Ui chnages updated

Updated changes based on review
Home button removed
Indented html
removed style with important keyword

grudger007 committed:
Merge branch 'master' into sprint4-ui
